client/state/jetpack-connect/reducer/jetpack-connect-authorize.js

```diff
@@ -75,7 +76,17 @@ export default function jetpackConnectAuthorize( state = {}, action ) {
 			} );
 		case JETPACK_CONNECT_QUERY_SET:
 			const queryObject = Object.assign( {}, action.queryObject );
-			return Object.assign( {}, buildDefaultAuthorizeState(), { queryObject: queryObject } );
+			const shouldAutoAuthorize = includes(
+				[ 'woocommerce-services-auto-authorize', 'woocommerce-setup-wizard' ],
+				queryObject.from
+			);
+			const additionalQueryArgs = shouldAutoAuthorize ? { autoAuthorize: true } : {};
+			return Object.assign(
+				{},
+				buildDefaultAuthorizeState(),
+				{ queryObject: queryObject },
```

Comment:
We can shorten `{ queryObject: queryObject }` to `{ queryObject }`

Reply:
👍 Fixed & rebased.